kyuubi-server/src/test/scala/org/apache/kyuubi/server/rest/client/BatchCliSuite.scala

```diff
@@ -72,6 +73,27 @@ class BatchCliSuite extends RestClientTestHelper with TestPrematureExit with Bat
                          |options:
                          |  verbose: true""".stripMargin
     Files.write(Paths.get(batchFile), batch_basic.getBytes(StandardCharsets.UTF_8))
+
+    val long_time_batch_basic = s"""apiVersion: v1
```

Comment:
The process may not be found during kill due to the process starting too fast or too slow, so I add a long-running job.
cc @turboFei